### PR TITLE
[4] Cleanup output html of email field

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue # .

### Summary of Changes

Cleanup of the output HTML from the email layout

### Testing Instructions

Go to a page with an email input - a good enable is the "I forgot my username" page on the frontend 

https://example.com/index.php/component/users/remind?Itemid=101

View the source. 

### Actual result BEFORE applying this Pull Request

```html
		<input
	type="email"
	inputmode="email"
	name="jform[email]"
	 class="form-control validate-email required"	id="jform_email"
	value=""
	 size="30"     autocomplete="email"    required  >
```

### Expected result AFTER applying this Pull Request

```html
<input type="email" inputmode="email" name="jform[email]" class="form-control validate-email required" id="jform_email" value="" size="30" autocomplete="email" required>
```

### Documentation Changes Required

None

### CODE REVIEW

File: layouts/joomla/form/field/email.php [hunk 1]
```diff
@@ -49,6 +49,13 @@
  */
 
 $attributes = array(
+	'<input',
```
Comment (Quy) [inline layouts/joomla/form/field/email.php]: Technically, this is not an attribute so maybe move it to the implode?

Comment (PhilETaylor) [reply layouts/joomla/form/field/email.php]: "Technically" haha

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 04338cafad067fc8f26c571e9ef41ab4dfd13575<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33475">issues.joomla.org/tracker/joomla-cms/33475</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 04338cafad067fc8f26c571e9ef41ab4dfd13575<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33475">issues.joomla.org/tracker/joomla-cms/33475</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33475">issues.joomla.org/tracker/joomla-cms/33475</a>.</sub>

**Comment by richard67** [comment]
Thanks!

**Pull request merged**